indico/modules/vc/plugins.py

```diff
@@ -185,7 +185,7 @@ def update_data_association(self, event, vc_room, event_vc_room, data):
         if event_vc_room.data is None:
             event_vc_room.data = {}
 
-    def update_data_vc_room(self, vc_room, data):
+    def update_data_vc_room(self, vc_room, data, is_new=False):
```

Comment:
:warning: This requires a change in the `vc_vidyo` plugin as well, unless we call it with `is_new=True` only if the method accepts this argument.

Reply:
Indeed!

Reply:
Solved, thanks!